lib/kino_db/connection_cell.ex

```diff
@@ -376,6 +376,6 @@ defmodule KinoDB.ConnectionCell do
   defp secrets() do
     System.get_env()
     |> Enum.filter(fn {k, _v} -> String.starts_with?(k, "LB_") end)
-    |> Enum.map(fn {k, _v} -> %{"label" => k, "value" => k} end)
+    |> Enum.map(fn {k, _v} -> %{"label" => String.trim_leading(k, "LB_"), "value" => k} end)
```

Comment:
```suggestion
    |> Enum.map(fn {k, _v} -> %{"label" => String.replace_prefix(k, "LB_", ""), "value" => k} end)
```

trim_leading would remove both `LB_` in `LB_LB_FOO`.